### PR TITLE
(PE-38292) Upgrade JRuby to 9.4.8.0

### PR DESCRIPTION
**Pull request by tlehman**
https://perforce.atlassian.net/browse/PE-38292

### CODE REVIEW

**Comment by justinstoller** [comment]
fwiw, I like to run the puppetserver unit tests with the update before merging. I'm doing that locally now. Will merge once those go green.

**Comment by donoghuc** [comment]
@tlehman can you edit the commit message to reference the ticket.

**Review by justinstoller — APPROVED** [review]
Tests pass locally for Puppet Server, I think this is good to go once the commit message is fixed up.

**Pull request merged**
